core/src/main/resources/lib/layout/breadcrumbBar.jelly

```diff
@@ -44,8 +44,16 @@ THE SOFTWARE.
               <j:forEach var="anc" items="${request.ancestors}">
                 <j:if test="${h.isModel(anc.object) and anc.prev.url!=anc.url}">
                   <li class="item">
+                    <!-- Workaround to set desired name to first breadcrumb (instead of Jenkins) -->
```

Comment:
Not: Would move into the choose block, otherwise context is unclear. Or just remove altogether.